Update README to also describe the Py 3.7 and Py 3.8 images

alexdutton opened the pull request:
I don't know whether the non-3.6 images have automated builds, so I've leftthat section alone, making sure it only refers to Py 3.6.

Resolves #33.

alexdutton committed:
Update README to also describe the Py 3.7 and Py 3.8 images

I don't know whether the non-3.6 images have automated builds, so I've left
that section alone, making sure it only refers to Py 3.6.

Resolves #33.

ppanero reviewed (approved):
LGTM! Thanks for this. Could you add the rolling builds of 3.7 and 3.8? the rules are exactly the same, but with the .7 and .8 instead of .6 :)

🚀 Might also need rebase, since I merged your other PR

alexdutton commented:
It currently says that `python3.6/Dockerfile` gets built and tagged with the git tag name. Should this be `3.6-{tag}` from now on, with the same pattern for other versions?

I'm going to hold off changing the README for now given the implication that all three Dockerfiles are tagged identically.

ppanero commented:
@alexsdutton hmm I see your point. However, I don't think they were ever `{tag}`... I think that was an error. If you see
- here: https://hub.docker.com/r/inveniosoftware/centos7-python/tags
- and here: https://hub.docker.com/r/inveniosoftware/centos8-python/tags

Which aligns with the current CI: https://github.com/inveniosoftware/docker-invenio/blob/master/.travis.yml#L24

Last but not least, the docker hub builds (I skipped 3.6 for simplicity but they are the same)

![Screenshot 2020-07-13 at 16 40 12](https://user-images.githubusercontent.com/6756943/87318140-39f5df80-c528-11ea-8ec6-b7a0c82e46b1.png)


Sorry for the late reply. If possible just edit the readme to reflect this. Otherwise I can merge :)

alexdutton committed:
Update automated build section of README for additional Py versions

Per @ppanero's comment¹, git tag builds were never configured, hence removing
that aspect of the README. As the Docker image tags are now included in the
table further up, I've removed the Dockerfile → image tag mapping from this
section entirely.

¹ https://github.com/inveniosoftware/docker-invenio/pull/36#issuecomment-657603389

alexdutton commented:
All ready again. Have also rebased, just in case.

ppanero commented on README.md:
```diff
@@ -40,20 +48,13 @@ The following environment variables has been set:
-
-This way, we will use tag `3.6` as latest for Python version and there is also the possibility to push tags for specific use cases such as pinning certain libraries or patches.
+Automated builds are configured using [Docker Hub builds](https://docs.docker.com/docker-hub/builds/), with each
+push to master initating a Docker Hub build that will be tagged per the table above.
```
```suggestion
push to master initating a Docker Hub build that will be tagged according to the Python version.
```

ppanero reviewed (approved):
Small suggestion. Otherwise LGTM. I tried rebasing and applying this fix myself... but due to permissions (did you allow edit from maintainers?) I cannot push to your branch :(

alexdutton commented:
@ppanero Rebased! :tada:
